changelog/fixed-find-incorrect-function-die.md

```diff
@@ -0,0 +1 @@
+`UnitInfo::get_function_dies()` sometimes retrieved the wrong function die, because of insufficient range filtering.
```

Comment:
The API isn't public, so this changelog entry isn't really valuable for the users IMO. Could you instead describe the observable error that got fixed?

Reply:
Done :)